pkg/signing/signing.go

```diff
@@ -0,0 +1,73 @@
+/*
+Copyright 2020 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package signing
+
+import (
+	"github.com/tektoncd/chains/pkg/signing/formats"
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+	versioned "github.com/tektoncd/pipeline/pkg/client/clientset/versioned"
+	"go.uber.org/zap"
+)
+
+const (
+	// ChainsAnnotation is the standard annotation to indicate a TR has been signed.
+	ChainsAnnotation = "chains.tekton.dev/signed"
+)
+
+// IsSigned determines whether a TaskRun has already been signed.
+func IsSigned(tr *v1beta1.TaskRun) bool {
+	signature, ok := tr.ObjectMeta.Annotations[ChainsAnnotation]
+	if !ok {
+		return false
+	}
+	return signature != ""
```

Comment:
Should we allow any non-empty string to signify that the taskrun is signed or only allow "true" since that's what we explicitly set?

Reply:
Good point. I'll tighten this up for now.